plugins/woocommerce/src/Internal/DataStores/Orders/OrdersTableDataStore.php

```diff
@@ -609,6 +609,15 @@ public function backfill_post_record( $order ) {
 			}
 		}
 		self::$backfilling_order_ids = array_diff( self::$backfilling_order_ids, array( $order->get_id() ) );
+
+		/**
+		 * Fired when the backing post record for an HPOS order is backfilled after an order update.
+		 *
+		 * @since 8.5.0
+		 *
+		 * @param \WC_Order $order The order object.
+		 */
+		do_action( 'woocommerce_hpos_post_record_backfilled', $order );
```

Comment:
I noticed that when I updated an order from within the order UI (change the order status, in my case), this hook fired 3 or 4 times. If the main use case for this hook is logging, it seems like this could add a lot of extra noise. So, a couple thoughts/questions here:

1. Should this routine really be running multiple times whenever an order is updated? (Maybe an exploration for a separate PR, if not)
2. Is there data we could add as another parameter to the hook here, or perhaps data already within the $order object, that could be used in a callback to further refine when a log entry actually gets written?